### PR TITLE
fix: retrieves correct guide url

### PR DESCRIPTION
**Pull request by FredericEspiau**
### Description

Before: guide link reached 404 error
After: retrieved the correct guide for each subsidiary

Internal reference: SCFRCA-1217

### CODE REVIEW

File: client/app/cda/details/cda-details.controller.js [hunk 1]
```diff
@@ -13,11 +13,9 @@ angular.module("managerApp")
         self.guides = {
             title: $translate.instant("cda_guide_title"),
             list: [{
-                name: $translate.instant("cda_guide_name"),
-                url: ovhDocUrl.getDocUrl("cloud/storage/ceph")
```
Comment (FredericEspiau) [inline client/app/cda/details/cda-details.controller.js]: This uses 2API. I've removed altogether as we're not supposed to use 2API if we can prevent it.

File: client/app/cda/details/cda-details.controller.js [hunk 2]
```diff
@@ -29,10 +27,22 @@ angular.module("managerApp")
             self.messageHandler = CloudMessage.subscribe("paas.cda", { onMessage: () => self.refreshMessage() });
         };
 
+        self.loadGuides = function () {
+            return OvhApiMe.v6()
+                .get().$promise
+                .then(me => {
+                    self.guides.list[0].url = URLS.guides.cda[me.ovhSubsidiary];
+                })
```
Comment (neolitec) [inline client/app/cda/details/cda-details.controller.js]: ovhDocUrl has been precisely made to prevent the usage of URLS.
Don't we have another choice?

Comment (FredericEspiau) [reply client/app/cda/details/cda-details.controller.js]: It's a 2API service and they are supposed to be replaced soonish so I don't know, I don't think we should bet on such things 😃

**Comment by jleveugle** [comment]
Hello @FredericEspiau,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request merged**